app/views/local_transaction/results.html.erb

```diff
@@ -1,6 +1,8 @@
 <%
   add_view_stylesheet("local-transaction")
 %>
+<% content_for :title do %>Pay your Council Tax: search result - GOV.UK<% end %>
```

Comment:
I think "Pay your Council Tax" is `publication.title`, so you could just have:
```
<% content_for :title do %>"#{publication.title}: t("formats.local_transaction.search_result")"<% end %>
```

Reply:
This sounds good to me, it appears this template is used for lots of different local transactions as well, so this would avoid adding the "Pay your Council Tax" to the title for the [Apply for free school meals](https://www.integration.publishing.service.gov.uk/apply-free-school-meals/edinburgh) page for example.

There does not appear to be a test in place to capture this, so will try and add something to this PR for this as well.